package.json

```diff
@@ -109,16 +109,16 @@
     "html-webpack-plugin": "^2.16.0",
     "if-env": "^1.0.4",
     "ip": "^1.1.5",
-    "joi": "^13.4.0",
+    "joi": "^13.3.0",
```

Comment:
are all these package version changes intentional?

Reply:
@iamigo I'm only seeing changes to view/rooms/app.js